packages/react/src/atoms/grid/base.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { ReactNode } from "react";
+import "../../../tailwind.css";
+import { useThemeContext } from "../../theme/ThemeProvider";
+import resolvedStyleProps from "../../utils/resolvedStyleProps";
+import LabelBase from "../labels/base";
+import {
+  LabelBaseProps,
+  GridBaseClasses,
+} from "../../../../interfaces/intefaces";
+
+export interface GridBaseProps {
+  content: JSX.Element[] | React.FC[] | string[];
+  variant?: string;
+  labelBaseProps?: LabelBaseProps;
+  gridBaseClasses?: GridBaseClasses;
+}
+
+const GridBase = (props: GridBaseProps) => {
+  const theme: any = useThemeContext();
+
+  const gridClassName = resolvedStyleProps(
+    "gridBaseClasses",
+    [
+      "gap",
+      "grid",
+      "size",
+      "border",
+      "margin",
+      "columns",
+      "padding",
+      "alignment",
+      "backgroundColor",
+    ],
+    props,
+    theme
+  );
+
+  return (
+    <>
+      {props.labelBaseProps && <LabelBase {...props.labelBaseProps} />}
```

Comment:
@ShreyDhyani I am not sure why Label is part of Grid. Because, if someone wants Label also, they would import label and grid and use it. I don't think we need label here.

Reply:
Will remove it @rjvim